util/test/demos/vk/vk_mesh_zoo.cpp

```diff
@@ -336,7 +336,7 @@ void main()
 
     VkPipeline stride0pipe = createGraphicsPipeline(pipeCreateInfo);
 
-    VkPipeline xfbpipe;
+    VkPipeline xfbpipe = NULL;
```

Comment:
Technically `NULL` is not necessarily a valid vulkan handle and this will error on some systems (though not sure anyone is trying to build this project there). This should be `VK_NULL_HANDLE`.

Reply:
Thanks for catching that. I'm mostly a DX developer myself. The change has been made.